ds_judgements_public_ui/templates/layouts/base.html

```diff
@@ -27,21 +27,65 @@
+                      </g>
+                    </svg>
+                  </a>
+                  <div class="beta-header__service">
+                    <h1 class="beta-header__service-title">Find case law<span class="beta-header__beta-label">BETA</span></h1>
+                    <p class="beta-header__helper-text">Judgments and decisions from 2001 onwards</p>
+                  </div>
+                </div>
+                <div class="govuk-header__content">
+                  <nav aria-label="Menu" class="govuk-header__navigation">
+                    <button type="button" class="govuk-header__menu-button govuk-js-header-toggle" aria-controls="navigation" hidden>
+                      Menu
+                    </button>
+                    <ul id="navigation" class="govuk-header__navigation-list">
+                      <li class="govuk-header__navigation-item govuk-header__navigation-item--active">
+                        <a class="govuk-header__link" href="{% url 'home' %}">
+                          Home
+                        </a>
+                      </li>
+                      <li class="govuk-header__navigation-item">
+                        <a class="govuk-header__link" href="{% url 'about_this_service' %}">
+                          About
+                        </a>
+                      </li>
+                      <li class="govuk-header__navigation-item">
+                        <a class="govuk-header__link" href="{% url 'how_to_use_this_service' %}">
+                          Help and guidance
+                        </a>
+                      </li>
+                    </ul>
+                  </nav>
+                </div>
+              </div>
+            </header>
             <div class="page-header__breadcrumb">
```

Comment:
Let's think about moving the breadcrumbs out of the header as it causes a jarring layout shift.